docs/src/test/java/io/improbable/snippet/SequenceExample.java

```diff
@@ -112,10 +114,26 @@ public Sequence buildSequenceTimeSeries() {
 
         Sequence sequence = new SequenceBuilder<Integer>()
             .withInitialState(dictionary)
+            .withIdentifyingNamespace("Keanu-Example")
             .count(5)
             .withFactory(factory)
             .build();
 
+        //We can now do a few things:
+        Vertex<?> seedVertex = sequence.getLastItem().get(x1Label);
+        BayesianNetwork network = new BayesianNetwork(seedVertex.getConnectedGraph());
+
+        //Within `network` our vertices will have the labels of the form:
+        //Keanu-Example.Sequence_Item_<<timestep>>.<<unique-hash>>.<<label>>
+        List<Vertex> allXVertices = network.getVerticesIgnoringNamespace(x1Label.getUnqualifiedName());
```

Comment:
needs an extra line of explanation e.g.

    // You can get all the vertices with a particular name, regardless of which SequenceItem they belong to.